joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 50

wheel_left_ctrl:
  type: velocity_controllers/JointVelocityController
  joint: wheel_left_joint

wheel_right_ctrl:
  type: velocity_controllers/JointVelocityController
  joint: wheel_right_joint